lib/src/package_analyzer.dart

```diff
@@ -54,6 +59,7 @@ class InspectOptions {
     this.isInternal = false,
     this.lineLength,
     this.analysisOptionsUri,
+    this.lockedVersions,
```

Comment:
Could we give this an empty map as a default, that would save a null check later.